docs/environment.rst

```diff
@@ -53,8 +53,9 @@ Details of the toml language subset
 CircuitPython behavior
 ----------------------
 
-CircuitPython will also read the environment to configure its behavior. Other
-keys are ignored by CircuitPython. Here are the keys it uses:
+CircuitPython will also read the environment to configure its behavior. Some keys are read at
+startup once and others are read on reload. If a reload doesn't change things, then try a restart.
+Other keys are ignored by CircuitPython. Here are the keys it uses:
```

Comment:
How about something like this?
```suggestion
CircuitPython will also read the environment to configure its behavior. Some keys are read at
startup once and others are read on reload (ctrl-D in the REPL). If a reload doesn't change things, then try a restart (a power cycle or pressing the reset button).
Other keys are ignored by CircuitPython. Here are the keys it uses:
```

Reply:
Ya, I'll make it consistent and say "reset".